FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -5526,7 +5561,196 @@ subroutine moist_bud2(im,ix,ix2,levs,me,kdt,grav,dtp,delp,rain, &
 
       end subroutine moist_bud2
 
+      subroutine moist_cv(initial_dynamics_q, physics_q, pressure_on_interfaces, &
```

Comment:
atmosphere.F90 uses the fmapz moist_cv, but this is a new one? I see the comment describing how it is different from the fv_mapz one, that is great! Would it be less confusing if the name were different, like moist_cv_nwat6 or moist_cv_simple? I am fine leaving it as is, it just confused me at first glance

Reply:
Indeed I like the idea of naming this something slightly different.  I think I'll go with your suggestion of `moist_cv_nwat6`.